### PR TITLE
Login Process

### PR DESCRIPTION
**Pull request by saraigyg**
sign up, sign in, sign out based on two different roles (admin and waiter) with firebase/firestore

### CODE REVIEW

File: my-bq-app/src/firebase/credentials.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// import function to initialize firebase app
```
Comment (PerlaDelAngel) [inline my-bq-app/src/firebase/credentials.js:1]: Remember to add this to your git ignore file (:

File: my-bq-app/src/pages/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import SelectVariants from "../components/SelectVariants";
+
+const auth = getAuth(firebaseApp);
+const firestore = getFirestore(firebaseApp);
+
+function Login () {
+    const [isRegistered, setIsRegistered] = useState(false);
+
+    async function userSignUp(email, password, role) {
+      const userInfo = await createUserWithEmailAndPassword(
```
Comment (PerlaDelAngel) [inline my-bq-app/src/pages/Login.jsx:16]: Would it be better to move the firebase functions to their own file? To separate responsabilities, also for testing purposes. If you do decide to move them, you could put them in a folder named lib like we've done in other projects :)

File: my-bq-app/src/pages/NotFoundPage.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```
Comment (PerlaDelAngel) [inline my-bq-app/src/pages/NotFoundPage.jsx:1]: Loving that you added a 'not found' page! Where are you calling this? I've never used it so I'm curious :o

**Review by PerlaDelAngel — APPROVED** [review]

File: my-bq-app/src/firebase/credentials.js (hunk 1 above)
Comment (saraigyg) [reply my-bq-app/src/firebase/credentials.js:1]: thank you Perla, yes I forgot.

File: my-bq-app/src/pages/Login.jsx (hunk 2 above)
Comment (saraigyg) [reply my-bq-app/src/pages/Login.jsx:16]: I already have a firebase folder, I think I could put it there, thank you :D

File: my-bq-app/src/pages/NotFoundPage.jsx (hunk 3 above)
Comment (saraigyg) [reply my-bq-app/src/pages/NotFoundPage.jsx:1]: I am calling it in another route inside App.js file like this: <Route path='*' element={<Not Found Page />}/> after importing it

**Pull request merged**